lib/ansible/modules/cloud/kubevirt/kubevirt_pvc.py

```diff
@@ -0,0 +1,279 @@
+      - Annotations attached to this object.
+      - U(https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/)
+    type: dict
+  labels:
+    description:
+      - Labels attached to this object.
+      - U(https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/)
+    type: dict
+  selector:
+    description:
+      - A label query over volumes to consider for binding.
+      - U(https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/)
+    type: dict
+  access_modes:
+    description:
+      - Contains the desired access modes the volume should have.
+      - More info: U(https://kubernetes.io/docs/concepts/storage/persistent-volumes#access-modes)
+    type: list
+  size:
+    description:
+      - How much storage to allocate to the PVC.
+    type: str
+    aliases:
+    - storage
+  storage_class_name:
+    description:
+      - Name of the StorageClass required by the claim.
+      - More info: U(https://kubernetes.io/docs/concepts/storage/persistent-volumes#class-1)
+    type: str
+  volume_mode:
+    description:
+      - volumeMode defines what type of volume is required by the claim. Value of Filesystem is implied when not included in claim spec. This is an alpha feature of kubernetes and may change in the future.
+    type: str
+  volume_name:
+    description:
+      - volumeName is the binding reference to the PersistentVolume backing this claim.
+    type: str
+  content_type:
+    description:
+      - Defines the content type of data being imported.
+      - I(kubevirt) denotes a virtual machine image. (Default if not specified)
+      - I(archive) denotes a tar archive.
+    choices:
+      - kubevirt
+      - archive
+  source:
+    description:
+      - If data is to be copied onto the PVC, specify the source.
+      - This uses the DataVolume source syntax: U(https://github.com/kubevirt/containerized-data-importer/blob/master/doc/datavolumes.md#https3registry-source)
+    type: dict
+
+extends_documentation_fragment:
+  - k8s_auth_options
+
+requirements:
+  - python >= 2.7
+  - openshift >= 0.8.2
+'''
+
+EXAMPLES = '''
+- name: Create a PVC and import data from an external source
+  kubevirt_pvc:
+    name: pvc1
+    namespace: default
+    size: 100Mi
+    access_modes:
+      - ReadWriteOnce
+    source:
+      http:
+        url: https://www.source.example/path/of/data/vm.img
+    # If the URL points to a tar.gz containing the disk image, ucomment the line below:
+    #content_type: archive
+
+- name: Create a PVC as a clone from a different PVC
+  kubevirt_pvc:
+    name: pvc2
+    namespace: default
+    size: 100Mi
+    access_modes:
+      - ReadWriteOnce
+    source:
+      pvc:
+        namespace: source-ns
+        name: source-pvc
+
+- name: Create a PVC ready for data upload
+  kubevirt_pvc:
+    name: pvc3
+    namespace: default
+    size: 100Mi
+    access_modes:
+      - ReadWriteOnce
+    source:
+      upload: yes
+
+#- name: Upload data to pvc
```

Comment:
Why to keep this commented example?

Reply:
Yeah, changed it to a short pointer to the `kubevirt_cdi_upload` module.